### ISSUE TITLE
4.0.0-beta.3 was released as "latest" instead of "next"

### ISSUE THREAD

**Issue by AviVahl** [jantimon/html-webpack-plugin#1104]
FYI @jantimon 
```sh
yarn info html-webpack-plugin dist-tags
yarn info v1.12.3
{ latest:
   '4.0.0-beta.3',
  beta:
   '2.6.5',
  next:
   '4.0.0-beta.2' }
Done in 0.59s.
```
I'm guessing you aimed for "next".

**Comment by AviVahl** [comment]
and for some reason there is a git submodule in the root named "html-webpack-plugin"

**Comment by jantimon** [comment]
Oh @AviVahl thanks a lot! 👍👍 fixed that and released 4.0.0-beta.4
